[TVJ] Tests and review

sorta opened the pull request:
Feature-complete travisci-vs-jenkins page, ready for Creative/Product review

Part of https://github.com/travis-ci/travis-web/pull/2293

Merges into epic branch

https://so-tvj-tests-and-feedback.test-deployments.travis-ci.com/travisci-vs-jenkins
https://so-tvj-tests-and-feedback.test-deployments.travis-ci.org/travisci-vs-jenkins

Nat-ess commented:
Hi! Thanks for this, looks great! 

Just a few comments:
- I agree with @artursmirnov, lets center the button for mobile.
- We could make the image a bit smaller so it's not so prominent in the mobile version?

sorta commented:
Ok I've implemented some changes according to feedback:

- Centered the contact button on mobile
- Adjusted the Choosing section image size and spacing on mobile
- The `.org` version of this page now redirects to `/`, as requested by marketing
- Rearranged and adjusted wording of Why section items
- Adjusted size and position of moz logo
- The `500` error seems to happen on `.org` but not `.com`. Since both the plans page and now this page are hidden on `.org`, it's probably fine for now? Might be worth looking into eventually though, if we ever want such a form to work on `.org`

It's once again ready for review @AlonaHlobina @pauladamgordon @Nat-ess

Nat-ess commented:
Looks good to me @sorta 👍, thanks for all your work!
